core/dslmcode/shared/drupal-7.x/modules/elmsln_contrib/cis_connector/modules/features/elmsln_core/elmsln_core.module

```diff
@@ -95,6 +95,29 @@ function elmsln_core_menu() {
 }
 
 /**
+ * Implements hook_menu_alter.
+ * Send user to the CPR to edit the global profile.
+ */
+function elmsln_core_menu_alter(&$items) {
+
+  if(variable_get('install_profile') != 'cpr') {
+    // Override the user edit handler for our purpose.
+    $items['user/%user/edit']['page callback'] = '_elmsln_people_user_edit_page';
+  }
+}
+
+/**
+ * Access callback for _elmsln_people_user_edit_page.
+ */
+function _elmsln_people_user_edit_page() {
+
+  // Setup a variable that can be changed if someone doesn't want to use the CPR for global profiles.
+  $elmsln_global_profile = variable_get('elmsln_global_profile', 'cpr');
+  // send user to the cpr we just defined and redirect them back to where they were.
+  _elmsln_core_remote_redirect($elmsln_global_profile, 'user', $GLOBALS['user']->uid, 'edit?destination=elmsln/redirect/' . variable_get('install_profile') . '/user/' . $GLOBALS['user']->uid);
```

Comment:
convention is variable_get('install_profile', 'standard') so there's a default even if it wouldn't do anything useful